src/app/admin/components/audio-file/associated-cases-table/associated-cases-table.component.ts

```diff
@@ -14,10 +14,16 @@ import { TableRowTemplateDirective } from '@directives/table-row-template.direct
 })
 export class AssociatedCasesTableComponent {
```

Comment:
Now this is re-used, we should probably move this file into a common directory, rather than `audio-file`?